source/dem/dem.cc

```diff
@@ -1556,7 +1600,7 @@ DEMSolver<dim>::solve()
                            triangulation,
                            particle_handler,
                            insertion_object,
-                           solids,
+                           solids_2d,
```

Comment:
maybe solid_surfaces and solide_volumes would be the better name here.

Reply:
I like it!